poc_config/openmrs/i18n/common/locale_pt.json

```diff
@@ -177,9 +177,9 @@
     "COMMON_PATERNAL_A": "Pai A",
     "COMMON_PATERNAL_B": "Pai B",
     "COMMON_VITALS": "Vitais",
-    "COMMON_TEMPERATURE_C": "Temperatura (C)",
-    "COMMON_WEIGHT": "Peso",
-    "COMMON_HEIGHT": "Altura/Estatura",
+    "COMMON_TEMPERATURE_C": "Temperatura (ºC)",
+    "COMMON_WEIGHT": "Peso (Kg)",
+    "COMMON_HEIGHT": "Altura/Estatura (cm)",
```

Comment:
Height should be (m) meters

Reply:
I think cm is right because we are using integers.

Reply:
Meters is the unit used by providers. I don't think we have space for changes here. If that is integer, then we should change to decimal

Reply:
@valvijo Can it be like this ?
![image](https://user-images.githubusercontent.com/4475142/31217952-0129de14-a9b9-11e7-8a74-344d5e63fe3f.png)

Reply:
:+1::+1::+1::+1: